samples/SimpleDataSource/Tables/Metadata/FileStatsMetadataTable.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Processing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SampleCustomDataSource.Tables.Metadata
+{
+    //
+    // This is a sample Metadata table for .txt files
+    // Metadata tables are used to expose information about the data being processed, not the actual data being processed.
+    // Metadata would be something like "number of events in the file" or "file size" or any other number of things that describes the data being processed.
+    // In this sample table, we expose three columns: File Name, Line Count and Word Count.
+    //
+
+    //
+    // In order for the CustomDataSourceBase to understand your metadata table, 
+    // add a MetadataTable attribute which denotes this table as metadata.
+    //
+
+    [Table]             
+    //
+    // Have the MetadataTable inherit the TableBase class
```

Comment:
nit: explicitly state "the **custom** TableBase class." This distinction is important because in the processor we inhereit from a ___Base" class that the SDK provides, so at first glance TableBase looks like an SDK thing.